### ISSUE TITLE
Will support for version 3 of the uptime monitor be made available? 

### ISSUE THREAD

**Issue by duffner** [LKDevelopment/laravel-uptime-monitor-api#8]

**Comment by LKDevelopment** [comment]
Since i doesn't use the package anymore i have no testing environment for this. But PRs are welcome!

**Comment by vesper8** [comment]
my fork adds compatibility with v3 as well as pulling in both current PRs and improved exception handling I got from another fork

**Comment by LKaemmerling** [comment]
@vesper8  if you want you could contribute this to this repo and get write access to it :)

**Comment by vesper8** [comment]
I don't know @LKaemmerling .. I've never maintained a public repo before.. totally unfamiliar with CI and how that works.. releases, tagging, and wouldn't even be sure whether these upgrades should go under a 3.0 version or stay on 2.0.. or how to even change that

I could submit a PR with all the changes and you could approve it? That's all it needs and it'll be good for another year or more

**Comment by LKaemmerling** [comment]
Yeah, this would be possible. If you want, i could show you everything which is needed for this :) OpenSource means teaching too :D

**Comment by vesper8** [comment]
I appreciate the generous offer. Even though I too have way too much on my plate at the moment, I'm inclined to agree, assuming it doesn't take too much of your time. How would we proceed?

**Comment by LKaemmerling** [comment]
First of all you should create a PR with your changes :) I will review it then and add my comments about style, tests and so on there.
